docker/dev/nodejs/Dockerfile

```diff
@@ -4,13 +4,12 @@ WORKDIR /code
 
 # Add dependencies
 ADD ./package.json /code
-ADD ./bower.json /code
 ADD ./gulpfile.js /code
 ADD ./.eslintrc /code
 ADD ./karma.conf.js /code
 
 # Install Prerequisites
-RUN npm install -g bower gulp
+RUN npm install -g yarn gulp
```

Comment:
Lets remove -g since we dont need it. Also, it throws error later if you are building from scratch